catalyst/utils/tests/test_torch.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from torch import nn
+
+from .. import torch as torch_utils
+
+
+def test_network_output():
+    """
+    Test for ``catalyst.utils.torch.get_network_output``
+    """
+    # case #0
+    net = nn.Identity()
+    assert torch_utils.get_network_output(net, (1, 20)).shape == (1, 1, 20)
+
+    # case #1
+    net = nn.Linear(20, 10)
+    assert torch_utils.get_network_output(net, (1, 20)).shape == (1, 1, 10)
+
+    # case #2
+    class Net(nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.net1 = nn.Linear(20, 10)
+            self.net2 = nn.Linear(10, 5)
+
+        def forward(self, x, y):
+            z = torch.cat((self.net1(x), self.net2(y)), dim=-1)
+            return z
+
+    net = Net()
+    assert torch_utils.get_network_output(net, (20,), (10,)).shape == (1, 15)
+
+    # case #3
+    class Net(nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.net = nn.Linear(20, 10)
+
+        def forward(self, input):
+            z = self.net(input["x"])
+            return z
+
```

Comment:
could we check here case like
```
def forward(self, *, x):
            z = self.net(x)
            return z
```
?

Reply:
I think no, as in this case we are going to test `**kwargs`, but the prototype of `torch.nn.Module.forward` is `forward(*input)` [pytorch docs](https://pytorch.org/docs/stable/nn.html#torch.nn.Module.forward)

Reply:
as4me, we should remove this functionality (`z = self.net(input["x"])`), 
or add support for `def forward(self, *, x):`
because, the case with `input["x"]` looks really hacky and complicated to understand